appcircle_audit_permission_changes/1.0.0/component.yaml

```diff
@@ -0,0 +1,17 @@
+platform: iOS
+buildPlatform:
+displayName: Audit Permission Changes
+description: "Permission Differance Checker"
+webUrl: https://github.com/appcircleio/appcircle-ios-permission-check-component
```

Comment:
### **3rd Review:**
- [ ] `AC_REPOSITORY_DIR` must be removed from README
- [ ] `AC_REPOSITORY_DIR` must be removed from component.yaml

**main.rb:**
- [ ] Instead using many times abort for env var, you can use this command:
```
def env_has_key(key)
  get_env_variable(key) || abort("Missing: #{key}")
end
```
- [ ] line 36: It should be checked and give error if `AC_REPOSITORY_DIR` not exist.
 - [ ] line 37: It should be checked and give error if `AC_PROJECT_PATH ` not exist.
 - [ ] Instead of using it twice, the code `ac_cache_label.gsub('/', '_')` should be assigned to the variable
 - [ ] `ac_cache_included_paths` is a path but it name like a area. And in the code on line 299 is written like an array. This variable must be a path not array
 - [ ] Is there a need to zip since only one txt file is cached?
 - [ ] Why we use `each do` in 321?
 - [ ] What is purpose of this line: 386?